doc/api.md

```diff
@@ -0,0 +1,120 @@
+# API
+
+## cartodb.DI.createDashboard(domObject, vizJson, callback(err, obj))
+Creates a new dashboard inside `domObject` based on `vizJson`.
+
+#### Arguments
+- domObject: dom object identifier where the dashboard will be inserted
+- vizJson: either a vizjson url or a vizjson object. See viz.json spec to see how to generate one
+- callback: it will be called when the dashboard is generated. First argument will be null if
+  everything was fine, an `Error` object otherwise. Second argument is a `Dashboard` object, see
+  below
+
+#### Example
+```js
+cartodb.deepInsights.createDashboard('#dashboard', vizJSONurl, function(err, dashboard) {
+  if (err) {
+    console.log('there was an error generating the dashboard');
+    return;
+  }
+  var map = dashboard.getMap()
+});
+```
+
+## cartodb.DI.Dashborbard
+
+This object contains a CartoDB dashboard, a map, and some widgets:
+
+##### getWidgets() -> Array of widgets
+##### getWidget(id) -> get widget
+##### getMap() -> returns a Map object
```

Comment:
Why are we providing a method to retrieve the `map` instead of the whole `vis`? If we return the `vis` object we can always refer people to the CartoDB.js docs so that they know what to do with that object.